docs/api-docs/provider-apis/tax/overview.mdx

```diff
@@ -143,6 +143,16 @@ BigCommerce sends requests for tax estimates on product item, shipping, and hand
 
 [View the Estimate Taxes API reference](/docs/rest-contracts/tax#estimate-taxes).
 
+### Performance
+
+Merchants expect fast tax estimate responses to facilitate a streamlined checkout experience for their shoppers. Hence performance is an essential consideration for a tax provider.
+
+To protect the shopper checkout experience from overly slow tax responses, tax estimate requests will time out after a set period of time. The default timeout threshold of four seconds is intended to support a wide variety of use cases (e.g. large tax estimate requests).
```

Comment:
👁️ Adjusted the wording here from `requests will timeout` to `requests will time out`, which I understand is more correct.